docs/source/app-dev/authorization.rst

```diff
@@ -7,124 +7,195 @@ Authorization
++-------------------------------------+----------------------------+--------------------------------------------------------+
+| TimeService                         | GetTime                    | public                                                 |
+|                                     +----------------------------+--------------------------------------------------------+
+|                                     | SetTime                    | participant_admin                                      |
++-------------------------------------+----------------------------+--------------------------------------------------------+
+| TransactionService                  | LedgerEnd                  | public                                                 |
+|                                     +----------------------------+--------------------------------------------------------+
+|                                     | All (except LedgerEnd)     | for each requested party p: canReadAs(p)               |
++-------------------------------------+----------------------------+--------------------------------------------------------+
+| UserManagementService               | All                        | participant_admin                                      |
+|                                     +----------------------------+--------------------------------------------------------+
+|                                     | GetUser                    | authenticated users can get their own user             |
+|                                     +----------------------------+--------------------------------------------------------+
+|                                     | ListUserRights             | authenticated users can list their own rights          |
++-------------------------------------+----------------------------+--------------------------------------------------------+
+| VersionService                      | All                        | public                                                 |
++-------------------------------------+----------------------------+--------------------------------------------------------+
+
+
+.. _access-token-formats:
+
+Access token formats
+********************
+
+Applications should treat access tokens as opaque blobs.
+However as an application developer it can be helpful to understand the format of access tokens to debug problems.
+
+All Daml ledgers represent access tokens as `JSON Web Tokens (JWTs) <https://datatracker.ietf.org/doc/html/rfc7519>`_,
+and there are two formats of the JSON payload in use by Daml ledgers.
+
+
+User access tokens
+==================
+
+Daml ledger that support participant user management also accept user access tokens.
+They are useful for scenarios where an application's rights change dynamically over the application's lifetime.
+
+User access tokens do not encode rights directly like custom Daml claims tokens.
```

Comment:
Forward reference to the custom Daml claims tokens header here.

Reply:
Overlooked that one. Fixing in another PR.

Reply:
https://github.com/digital-asset/daml/pull/12534